### DISCUSSION TITLE
Usage with the Divi Builder

### DISCUSSION THREAD

**Discussion by DrCoolzic** [KevinBatdorf/code-block-pro#243] [General]
Hello I love your plugin !
I am using the Divi Theme/Builder and I have not found a way to use code-block-pro on page edited with Divi ?
I am new to wordpress divi and may be the question is stupid, but I would like to know if you have tried your plugin on a divi theme page?
Thanks
Jean

**Comment by KevinBatdorf** [comment]
Hey, it's not a stupid question by any means, but it's also not possible as far as I'm aware. My code requires the WordPress block editor, and Divi (if i remember correctly), disables that uses their own completely separate editor. It would be impossible to add support without writing a brand new plugin for Divi. 

Since you're new to WordPress, have you tried using the built in editor? You might find it covers all your needs these days.

**Comment by DrCoolzic** [comment]
Thanks for your feedback. With the build in editor your module works fine but I bought Divi because it has a lot of nice features that help develop nice pages more easily. I'll find a solution for documenting my code.
